### DISCUSSION TITLE
What do I need to export from Google Takeout to have Google location data (instead of using EXIF)?

### DISCUSSION THREAD

**Discussion by erkexzcx** [simulot/immich-go#379] [Q&A]
Title. Is it enough to export Google photos only during Takeout, or should I also select Google maps data during Takeout?

Related to these release notes: https://github.com/simulot/immich-go/releases/tag/0.19.0

++ what do I do if I already imported pictures? Should I delete them from Immich first, or will immich-go automatically update location data?

**Comment by simulot** [comment]
Immich-go force the photo's date and GPS with data found in the JSON files, but only for new files.

I have yet developed a use "google map" GPS feature...  many later.
At the moment, the sole solution is to delete the concerned immich photos, and re-upload them from the takeout.

For a future versions, I can implement the  GPS coordinates update of existing immich photos based on takeout json. Does it make some sense?
